### ISSUE TITLE
Keeps complaining about missing packages when installing

### ISSUE THREAD

**Issue by S3j5b0** [nexB/scancode-toolkit#2489]
### Description

Hi, im trying to get and run scancode for scanning my golang application. I'm on a ubuntu 20 machine. I am trying to follow the recommended path decribed in your readme.
I am using the latest release that I could find here.

First I go and download the latest release:

`scancode-toolkit-21.3.31_py36-linux.tar.xz`

I then unpack it:

`tar -xvf scancode-toolkit-21.3.31_py36-linux.tar.xz`

I also run this command, as described in the prereq page of the documentation:
`
sudo apt install python3.8-dev bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev libpopt0`

I then run the executeable, which renders the following:

```
./scancode
* Configuring ScanCode for first use...
ERROR: Could not find a version that satisfies the requirement pyahocorasick<1.5,>=1.4 (from scancode-toolkit)
ERROR: No matching distribution found for pyahocorasick<1.5,>=1.4
```

now, if I run ' ./configure --clean' and run scancode again, the same happens, just with another dependency.

Ok, so I try installing all dependencies from the requirements.txt file:

`pip3 install -r ./requirements.txt 
`

This however has no effect, the still issue persists. 
I then try installing them with the pip versions in the /bin folder:

`./bin/pip3 install -r ./requirements.txt 
`
and the same problem persists, also for the pip3.6  and pip files
What is the issue here?

### System configuration

* Ubuntu 20.04
* -21.3.31

**Comment by pombredanne** [comment]
@S3j5b0 The application tarball `scancode-toolkit-21.3.31_py36-linux.tar.xz` tarball contains only pre-built 3.6 Python dependencies and is designed to work only with Python 3.6 for now so you will not get something working with your Python 3.8. 
(I created #2491 as a related ticket)

The way to go would be instead:
A) From git: https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install
B) Using pip: https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#pip-install

In all cases, start with and install of  the pre-requisites: https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#before-installing
`sudo apt install python3.6-dev bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev libpopt0`

You may need too for a git or pip installation:
`sudo apt-get install build-essential`

For a pip installation, create a virtualenv and install it:
```
mkdir tmp
wget -O tmp/virtualenv.pyz https://bootstrap.pypa.io/virtualenv/virtualenv.pyz
python3 tmp/virtualenv.pyz tmp
source tmp/bin/activate
pip install scancode-toolkit
```

For a git clone:
```
git clone https://github.com/nexB/scancode-toolkit
cd scancode-toolkit
./configure
```

**Comment by S3j5b0** [comment]
Hmmm, I just assumed I could use 3.8 because the readme says that one can use 3.6+ 

I have now installed via pip.

running 'scancode' gives me this view:

```
/home/ask/.local/lib/python3.8/site-packages/cluecode/copyrights.py:3122: FutureWarning: Possible set difference at position 3
  remove_tags = re.compile(
/home/ask/.local/lib/python3.8/site-packages/nltk/tag/sequential.py:532: FutureWarning: Possible nested set at position 7
  self._regexs = [(re.compile(regexp), tag,) for regexp, tag in regexps]
Usage: scancode [OPTIONS] <OUTPUT FORMAT OPTION(s)> <input>...
Try the 'scancode --help' option for help on options and arguments.

Error: Invalid value: At least one input path is required.
```

Which doesnt make much sense to me

**Comment by pombredanne** [comment]
@S3j5b0 
the  `FutureWarning: ...` thing should not be displayed. But the usage help looks fine to me is this still a problem?

**Comment by robguinness** [comment]
> @S3j5b0
> the `FutureWarning: ...` thing should not be displayed. But the usage help looks fine to me is this still a problem?

I am also seeing the same FutureWarnings. Should I open a separate issue for this?

**Comment by robguinness** [comment]
Could you please explain why you closed this issue?

**Comment by pombredanne** [comment]
@S3j5b0 not sure either why you closed ... I guess that your main issue is resolved? let me reopen to deal with the FutureWarning messages. They are innocuous for now, but annoying

**Comment by pombredanne** [comment]
The warning has been solved with #2686 and https://github.com/nexB/scancode-toolkit/commit/5c5235c10c96e5c1baa1ac938a8f44568f7432c7

**Comment by pombredanne** [comment]
Closing this now. Please report using the latest 31.0.0rc1 to double validate this!
Thanks
